elasticsearch/src/main/java/org/hibernate/search/elasticsearch/cfg/ElasticsearchEnvironment.java

```diff
@@ -223,6 +224,21 @@
 	 */
 	public static final String SCROLL_TIMEOUT = "elasticsearch.scroll_timeout";
 
+	/**
+	 * Property for specifying the behaviour when Elasticsearch encounters a previously unknown field in a document.
+	 * <p>
+	 * Possible values are:
+	 * <p>
+	 * <ul>
+	 *   <li>{@code true}: Add new fields dynamically</li>
+	 *   <li>{@code false}: Ignore new fields</li>
+	 *   <li>{@code strict}: Throw an exception if an unknown field is encountered</li>
```

Comment:
Same comment on "unknown field"; I might be wrong but I suspect it's not very clear.